docs/config/kafka.md

```diff
@@ -22,16 +18,16 @@ servers:
-| EVENTGATEWAY_KAFKA_PROXY_MESSAGE_VALIDATION_ENABLED | boolean | Enable or disable validation of Kafka messages                                                                                                                                                                                                       | `true`  | No                                | `true`, `false`                                                                                                                                                                                              |
-| EVENTGATEWAY_KAFKA_PROXY_EXTRA_FLAGS                | string  | Advanced configuration. Configure any flag from [here](https://github.com/grepplabs/kafka-proxy/blob/4f3b89fbaecb3eb82426f5dcff5f76188ea9a9dc/cmd/kafka-proxy/server.go#L85-L195). Multiple values can be configured by using pipe separation (`\|`) | -       | No                                | `tls-enable=true\|tls-client-cert-file=/opt/var/service.cert\|tls-client-key-file=/opt/var/service.key`                                                                                                        |
\ No newline at end of file
+## Advanced configuration
+Some advanced configuration is only available through environment variables.
+
+| Environment variable                        | Type   | Description                                                                    | Default | Required | examples                         |
+|---------------------------------------------|--------|--------------------------------------------------------------------------------|---------|----------|----------------------------------|
+| EVENTGATEWAY_KAFKA_PROXY_BROKER_FROM_SERVER | string | When set, only the specified server will be considered instead of all servers. | -       | No       | `name-of-server1`, `server-test` |
+| EVENTGATEWAY_KAFKA_PROXY_MESSAGE_VALIDATION_ENABLED | boolean | Enable or disable validation of Kafka messages                                                                                                                                                                                                       | `true`  | No       | `true`, `false`                                                                                         |
+| EVENTGATEWAY_KAFKA_PROXY_EXTRA_FLAGS                | string  | Advanced configuration. Configure any flag from [here](https://github.com/grepplabs/kafka-proxy/blob/4f3b89fbaecb3eb82426f5dcff5f76188ea9a9dc/cmd/kafka-proxy/server.go#L85-L195). Multiple values can be configured by using pipe separation (`\|`) | -       | No       | `tls-enable=true\|tls-client-cert-file=/opt/var/service.cert\|tls-client-key-file=/opt/var/service.key` |
```

Comment:
I don't see any code where you read these values from envs and save them to the `KafkaProxy` struct. Are you using any env parsing library or are these envs will be removed?

Reply:
I'm using https://github.com/kelseyhightower/envconfig which is pretty common nowadays.

Reply:
One thing puzzles me. Is a variable `EVENTGATEWAY_KAFKA_PROXY_MESSAGE_VALIDATION_ENABLED` well read if it has suffix `_ENABLED` and in the structure the variable is without this suffix (MessageValidation)?

Reply:
The value we are setting here is the `Enabled` struct property of `MessageValidation`. You can see it here: https://github.com/smoya/event-gateway/blob/chore/messagePackage/config/kafka.go#L28. 

`envconfig` works by generating flags with the name based on the structs hierarchy. Let me explain with the example `EVENTGATEWAY_KAFKA_PROXY_MESSAGE_VALIDATION_ENABLED`:

- `EVENTGATEWAY` is the prefix for the app. This is set manually when starting `envconfig`.  See https://github.com/smoya/event-gateway/blob/chore/messagePackage/main.go#L22
- `KAFKA_PROXY` is the `KafkaProxy` property for the struct `Config`. See it here: https://github.com/smoya/event-gateway/blob/chore/messagePackage/config/config.go#L14
- `MESSAGE_VALIDATION` is the `MessageValidation` property for the struct `KafkaProxy`. You can see it here: https://github.com/smoya/event-gateway/blob/chore/messagePackage/config/kafka.go#L22
- `ENABLED` is the `Enabled` property for the struct `MessageValidation`. You can see it here: https://github.com/smoya/event-gateway/blob/chore/messagePackage/config/kafka.go#L28